src/main/java/com/example/bistro/bistroapp/repository/ProductRepository.java

```diff
@@ -0,0 +1,16 @@
+package com.example.bistro.bistroapp.repository;
+
+import com.example.bistro.bistroapp.entity.Product;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+
+import java.util.List;
+
+public interface ProductRepository extends JpaRepository<Product, Long> {
+
+    @Query("SELECT p FROM Products p LEFT JOIN p.orders o GROUP BY p ORDER BY COUNT(o) DESC")
```

Comment:
How will this method work to get 10 most wanted products if we don't have any products in db?